### ISSUE TITLE
[Question] How many levels of header grouping are possible?

### ISSUE THREAD

**Issue by prenaut41** [react-tools/react-table#715]
I tried creating a ReactTable with two levels of header grouping above the column names (such that there would be three header rows), but only the top two levels of column (header) objects were displayed, with the 3rd level (the level for the primary column name) being dropped.  

I got my table working by removing the top-most level of header grouping so that there are only two levels now for headers.  

I'm not a big fan of the higher-layers of column grouping (essentially grouping the groups) that  my user wants to do, but is it even possible to have more than two levels for the headers with only one level of header groups without some major, time-consuming work-around?

**Comment by gary-menzel** [comment]
TL;DR; Only 2 - no way to change this. If the requirement is a deal breaker you'll need to use a different table component.

The setup is done during the model state creation and is not recursive (i.e. it only looks at the first nested set of columns). While it appears to be "nested" it sort of works backwards (in the sense that it checks to see if the `columns` are wrapped in a "group" and creates an extra row above the columns for the grouping).  The columns are the key data control item and the spanned columns are just a decoration.
